packages/genesys-spark-components/src/components/beta/gux-month-picker/gux-month-calendar/gux-month-list/gux-month-list-item/gux-month-list-item.tsx

```diff
@@ -50,7 +50,6 @@ export class GuxMonthListItem {
           <button
             class={{ 'gux-selected': this.selected }}
             type="button"
-            tabIndex={-1}
```

Comment:
It seems like this code I removed was initially here for a reason I'd imagine so I'm going to track down when and why this code was added in the first place.

Reply:
It has a tab index of -1 because it was designed to use arrow navigation instead of tabbing through each of the months. Unless we have a reason to change the behavior here, I think the tabindex of -1 is intended behavior